### PR TITLE
AEA-2649 Log signature verification errors

### PR DESCRIPTION
**Pull request by MathewArnoldIW**

### CODE REVIEW

File: coordinator/src/services/communication/tracker/tracker-response-builder.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ const createTrackerResponse = (
 ): TrackerResponse => {
   const prescription = extractPrescription(spineResponse.body, logger)
   if (prescription) {
-    logger.info(`Successfully extracted prescription ${prescription.id}`)
+    logger.info(`Successfully extracted prescription ${prescription.id._attributes.root}`)
```
Comment (MathewArnoldIW) [inline coordinator/src/services/communication/tracker/tracker-response-builder.ts:56]: Trivial unrelated change, just a logging error found while doing this ticket's work

**Review by TomSmithCoding — APPROVED** [review]
